code-samples/oauth/simple-oauth-webserver/index.js

```diff
@@ -14,18 +13,28 @@ http.createServer((req, res) => {
 
 	if (urlObj.query.code) {
 		const accessCode = urlObj.query.code;
-		const data = new FormData();
+		const data = {
+			client_id: 'your client id',
+			client_secret: 'your client secret',
+			grant_type: 'authorization_code',
+			redirect_uri: 'your redirect uri',
+			code: accessCode,
+			scope: 'the scopes',
+		};
 
-		data.append('client_id', 'your client id');
-		data.append('client_secret', 'your client secret');
-		data.append('grant_type', 'authorization_code');
-		data.append('redirect_uri', 'your redirect url');
-		data.append('scope', 'your scopes');
-		data.append('code', accessCode);
+		// convert to application/x-www-form-urlencoded
+		function formUrlEncode(data) {
```

Comment:
Could you use [require('querystring').stringify(obj)](https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options) instead of that?